### ISSUE TITLE
Searching Brave’s browser history errors with ‘Database is locked’

### ISSUE THREAD

**Issue by adamsilver** [raycast/extensions#4494]
### Extension

https://www.raycast.com/crisboarna/browser-history

### Description

When I search Brave’s browser history it gives me the following error in the logs:

> Error: Error: database is locked
> at Object.current (/Users/adamsilver/.config/raycast/extensions/efd37bf6-5682-4037-b32b-eef31c38a247/search-history.js:8:3284)
> at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

See gif recording:
![screenshot_2023-01-28_at_11 47 07](https://user-images.githubusercontent.com/37163/215435249-fb0afee1-1873-49cc-9d36-77014f97421f.gif)

### Steps To Reproduce

Given I’m using my MacOS Monterey 12.0.1
When I open Raycast
And I type run the ‘Browser History’ command
Then Raycast does not show any results
And it shows an error as described in the description

**Comment by benqqqq** [comment]
Hi, 
I encountered the same problem in Microsoft Edge, which also uses an SQLite database to store browsing history information at `~/Library/Application\ Support/Microsoft\ Edge/<Profile>/History`. (in mac environment)
I guess this error happen when `getHistoryQuery` was called, and the sqlite database is locked. (https://github.com/raycast/extensions/blob/main/extensions/microsoft-edge/src/hooks/useHistorySearch.tsx). 

I'm not sure if the locking behavior is normal or not because a few weeks ago, I used another computer and it worked fine!

I also read an article that said Chrome's history SQLite database is locked while Chrome is running, so I tried closing Microsoft Edge, and the Raycast search history went back to normal.

By the way, this issue might have the same root cause: https://github.com/raycast/extensions/issues/5274.
